services/scenario-manager/grpcclient/clients_test.go

```diff
@@ -102,6 +119,7 @@ func TestTopologyClient(t *testing.T) {
 			if err != nil && errors.Is(err, test.err) {
 				t.Error("error: expected", test.err, "received", err)
 			}
+			log.Printf("responseTopo: %s", proto.MarshalTextString(res))
```

Comment:
Where is the log printed to?

Reply:
It will print to console for now. I have created a logger in the code, it will print the log to syslog if specify the parameter in the config.